### PR TITLE
Update use-the-delist-portal-to-remove-yourself-from-the-office-365-b…

### PR DESCRIPTION
**Pull request by hudish**
…locked-senders-lis.md

This proposed content change has been reviewed and verified by dhagarwal@microsoft.com.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis.md [hunk 1]
```diff
@@ -69,6 +69,14 @@ where  _IP address_ is the IP address of the computer on which the mail server r
 
 See [Create safe sender lists in EOP](create-safe-sender-lists-in-office-365.md) and [Outbound spam protection in EOP](outbound-spam-controls.md) to prevent an IP from being blocked.
 
-## More information
+### What about error code 5.7.511?
+ 
+When there's a problem delivering an email message that you sent, Microsoft 365 or Office 365 sends an email to let you know. The email you receive is a delivery status notification, also known as a DSN or bounce message. The most common type is called a non-delivery report (NDR) and they tell you that a message wasn't delivered. In certain situations, Microsoft must conduct additional investigations against traffic from your IP, and if you’re receiving the NDR code 5.7.511, you **will not** be able to use the delist portal.
+ 
+>	550 5.7.511 Access denied, banned sender[xxx.xxx.xxx.xxx]. To request removal from this list please forward this message to delist {at} messaging.microsoft.com. For more information please go to http://go.microsoft.com/fwlink/?LinkId=526653.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis.md]:
```suggestion
>	550 5.7.511 Access denied, banned sender[xxx.xxx.xxx.xxx]. To request removal from this list, forward this message to delist@messaging.microsoft.com. For more information, go to http://go.microsoft.com/fwlink/?LinkId=526653. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hudish, kindly commit the suggested changes. Many thanks.

**Comment by hudish** [comment]
I don't believe I have access to commit directly, can you help or clarify what I should do next?

**Review by MSFTTracyP — APPROVED** [review]
Thanks!

**Comment by MSFTTracyP** [comment]
@Dansimp Does this need further action from me?

**Pull request merged**